Source/WTF/wtf/PlatformEnable.h

```diff
@@ -585,6 +585,10 @@
 #define ENABLE_XSLT 1
 #endif
 
+#if !defined(FILE_SYSTEM_WRITABLE_STREAM)
+#define ENABLE_FILE_SYSTEM_WRITABLE_STREAM 0
```

Comment:
Doesn't this break the tests?

Reply:
oh yeah, I'm waiting bots to tell me which tests will fail.